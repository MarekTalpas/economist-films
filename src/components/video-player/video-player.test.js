// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player';

test('VideoPlayer renders correctly', () => {
  const tree : string = renderer.create(<VideoPlayer location={{ query: { id: 5 } }} />).toJSON();
  expect(tree).toMatchSnapshot();
});
