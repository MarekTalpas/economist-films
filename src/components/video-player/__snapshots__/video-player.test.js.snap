exports[`test VideoPlayer renders correctly 1`] = `
<div
  className="video-player">
  <p
    className="video-player__id">
    id: 
    5
  </p>
  <a
    onClick={[Function]}
    style={Object {}}>
    <== Back
  </a>
</div>
`;
